<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QPushButton" name="pushButtonBuscar">
    <property name="geometry">
     <rect>
      <x>610</x>
      <y>50</y>
      <width>81</width>
      <height>24</height>
     </rect>
    </property>
    <property name="text">
     <string>BUSCAR</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButtonOrganizar">
    <property name="geometry">
     <rect>
      <x>260</x>
      <y>80</y>
      <width>80</width>
      <height>24</height>
     </rect>
    </property>
    <property name="text">
     <string>ORGANIZAR</string>
    </property>
   </widget>
   <widget class="QComboBox" name="comboBox">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>80</y>
      <width>151</width>
      <height>24</height>
     </rect>
    </property>
    <item>
     <property name="text">
      <string>LISTAR POR CURSO</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>LISTAR POR DISCIPLINA</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>LISTAR POR TURMA</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>MOSTRAR ALUNOS</string>
     </property>
    </item>
   </widget>
   <widget class="QTextEdit" name="textEdit">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>120</y>
      <width>591</width>
      <height>241</height>
     </rect>
    </property>
   </widget>
   <widget class="QTextEdit" name="textEdit_referencia">
    <property name="geometry">
     <rect>
      <x>350</x>
      <y>80</y>
      <width>341</width>
      <height>31</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>50</y>
      <width>503</width>
      <height>24</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>-2</x>
      <y>-5</y>
      <width>801</width>
      <height>581</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap resource="Arquivos.qrc">:/ArquivosFotos/Arquivos/ArquivosFotos/PUC.jpg</pixmap>
    </property>
   </widget>
   <zorder>label</zorder>
   <zorder>pushButtonBuscar</zorder>
   <zorder>pushButtonOrganizar</zorder>
   <zorder>comboBox</zorder>
   <zorder>textEdit</zorder>
   <zorder>textEdit_referencia</zorder>
   <zorder>lineEdit</zorder>
   <zorder>comboBox</zorder>
   <zorder>pushButtonOrganizar</zorder>
   <zorder>textEdit_referencia</zorder>
   <zorder>lineEdit</zorder>
   <zorder>pushButtonBuscar</zorder>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources>
  <include location="Arquivos.qrc"/>
 </resources>
 <connections/>
</ui>
